GPU/Workflow/src/GPUWorkflowSpec.cxx

```diff
@@ -223,7 +223,7 @@ void GPURecoWorkflowSpec::init(InitContext& ic)
     mConfig->configWorkflow.inputs.setBits(GPUDataTypes::InOutType::TRDTracklets, true);
     mConfig->configWorkflow.steps.setBits(GPUDataTypes::RecoStep::TRDTracking, true);
   }
-  if (mSpecConfig.runITSTracking) {
+  if (mSpecConfig.runITSTracking || mSpecConfig.runITS3Tracking) {
```

Comment:
I would do this somewhat differently. Instead of having somewhat redundant and exclusive runITSTracking and runITS3Tracking, I'd keep runITSTracking to steer to run ITS tracking, and then have a separate flag ITSTrackingITS3 (or whatever name you like), to indicate that it will run the ITS3 version.

Reply:
Thanks for the review and indeed your suggestion is much cleaner :)